CppCoreGuidelines.md

```diff
@@ -21972,23 +21972,23 @@ Never allow an error to be reported from a destructor, a resource deallocation f
 
         void test(string& s)
         {
-            Nefarious n;          // trouble brewing
+            Nefarious n;          // trouble waiting to happen
```

Comment:
This replaces a common idiom with something very meh

Reply:
changed more, plz check